tasks/build_tags.py

```diff
@@ -1,12 +1,13 @@
 """
 Utilities to manage build tags
 """
+
 # TODO: check if we really need the typing import.
 # Recent versions of Python should be able to use dict and list directly in type hints,
 # so we only need to check that we don't run this code with old Python versions.
+from __future__ import annotations
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Is this a shared practice to import from `__future__`?

Reply:
Yes it is very common. You can find more info about it [here](https://docs.python.org/3/library/__future__.html)